Model/Export/Consumer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace StoreKeeper\StoreKeeper\Model\Export;
+
+use Psr\Log\LoggerInterface;
+use StoreKeeper\StoreKeeper\Model\ProductExporter;
+
+class Consumer
+{
+    private LoggerInterface $logger;
+    private ProductExporter $productExporter;
+
+    public function __construct(
+        LoggerInterface $logger,
+        ProductExporter $productExporter
+    ) {
+        $this->logger = $logger;
+        $this->productExporter = $productExporter;
+    }
+
+    /**
+     * @param string $request
+     * @return void
+     */
+    public function process($request): void
+    {
+        try {
+            $data = json_decode($request, true);
+            if ($data['entity'] == 'catalog_product') {
+                $this->productExporter->exportProductToCsv();
+            }
```

Comment:
nothing processd is an error in my opinion

Reply:
@ikulis Same as above - in core exporter if nothing to export - file just does not appears in files list, without any gui messages